conf/messages.en

```diff
@@ -466,6 +466,12 @@ confirmation.referenceNumber = Your Reference Number
 confirmation.whatHappensNext.heading = What happens next
 confirmation.whatHappensNext.p1 = We’ve sent your application to the NDRC team.
 confirmation.whatHappensNext.p2 = They will contact you either to confirm your registration, or to ask for more information.
+confirmation.improveService.heading = Help us improve our services
+confirmation.improveService.p1 = We use your feedback to make our services better.
+confirmation.improveService.link.text1 = What did you think of this service?
+confirmation.improveService.link.tab = ?newTab=true
+confirmation.improveService.link.text2 = (takes 30 seconds)
```

Comment:
@somaraja think we need to change this to 'takes one minute'. Ask Alex for Confirmation.

Reply:
@lmugnai this has been corrected to 1 minute after I confirmed with Alex.